pydantic/tools.py

```diff
@@ -1,12 +1,18 @@
+from __future__ import annotations
```

Comment:
Just a nit: was this necessary to get tests passing? If not I would have tables it

Reply:
I think what happened was, at some point I was importing something for type checking purposes that was defined in an `if TYPE_CHECKING:` block in its file, which was super annoying. Looks like that's gone now, but it seemed reasonable to keep it like this.

Reply:
Yeah no worries that’s a good enough reason. I only ask because I’ve done it accidentally